include/swift/Runtime/TracingCommon.h

```diff
@@ -33,7 +33,8 @@ static inline bool shouldEnableTracing() {
     return false;
   if (__progname && (strcmp(__progname, "logd") == 0 ||
                      strcmp(__progname, "diagnosticd") == 0 ||
-                     strcmp(__progname, "notifyd") == 0))
+                     strcmp(__progname, "notifyd") == 0) ||
```

Comment:
There's a mistake here in the parenthesis placement - I'm fixing it in https://github.com/apple/swift/pull/72612.

Reply:
@tshortli The stray paren is also in #72593; presumably you'll cherry pick #72612 to the `release/6.0` branch to fix?

Reply:
Yes will cherry pick.

Reply:
Thank you!!